### DISCUSSION TITLE
Can we create a Win10 Start Menu Theme?

### DISCUSSION THREAD

**Discussion by jmlindemann** [Open-Shell/Open-Shell-Menu#1893] [General]
I know that this app was created originally for bringing back the classic start menu into Windows 10\11. But I'd like to see a theme based on the WIndows10 start menu added. 
Having been forced into Windows11 (new computer), I became quite accustomed to the Win10 start menu. Not so much for the listing of apps on the left side, (though it was used) but more for the custom listing of app shortcuts, sections, and groupings I created on the right side.
Windows11 seems to want to force a section on recommendations and limit my space.  I'd just like to have the Win10 menu system back again.
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/64678443/0d80f0a7-28d8-4d8c-836d-bfd5d986723a)

**Comment by mstick99** [comment]
Might look into Stardock WindowBlinds if no Open-Shell solution.

**Reply by DerellLicht** [reply]
Also, WindowBlinds (which I've been using since 1999) does not actually skin the Start Menu any more; you have to separately purchase their Start11 program for that purpose... and BTW, Start11 does not actually support WinXP-style cascading menus for "All Programs" menu... 

They *did* recently add a hack to Start11 to support cascading "All Programs", but it really *is* a hack, and lacks major functions that Open Start Menu has had for decades...

**Comment by jmlindemann** [comment]
Thanks for the suggestion. I would go maybe $10 or $15…. But $45, no. I just want to revamp the start menu. That’s a bit high for just that one want

Sent from my iPhone

On Apr 17, 2024, at 8:06 PM, mstick99 ***@***.***> wrote:

﻿

Might look into Stardock WindowBlinds if no Open-Shell solution.
